src/main/java/com/linkedin/kmf/services/configs/CommonServiceConfig.java

```diff
@@ -17,7 +17,7 @@
   public static final String ZOOKEEPER_CONNECT_DOC = "Zookeeper connect string.";
 
   public static final String BOOTSTRAP_SERVERS_CONFIG = CommonClientConfigs.BOOTSTRAP_SERVERS_CONFIG;
-  public static final String BOOTSTRAP_SERVERS_DOC = CommonClientConfigs.BOOTSTRAP_SERVERS_DOC;
+  public static final String BOOTSTRAP_SERVERS_DOC = CommonClientConfigs.BOOSTRAP_SERVERS_DOC;
```

Comment:
Uh? This looks like a typo

Reply:
@mpetazzoni yes that is for 0.9.0.1 version of org.apache.kafka.clients.CommonClientConfigs